src/main/java/net/fabricmc/tinyremapper/MappedIterator.java

```diff
@@ -0,0 +1,28 @@
+package net.fabricmc.tinyremapper;
+
+import java.util.Iterator;
+import java.util.function.Function;
+
+public final class MappedIterator<F, T> implements Iterator<T> {
```

Comment:
What is the point of this? Seems unused?

Reply:
it's unused now yea, forgot to remove it